### ISSUE TITLE
pa11y tool

### ISSUE THREAD

**Issue by tnguyen14** [aduggin/accessibility-fails#1]
Would you consider adding a column for http://pa11y.org/? I'm really new to a11y testing and was wondering which tool to use. Your comparison table is really helpful, but I noticed that it doesn't have pa11y.

**Comment by aduggin** [comment]
Sorry - just seen this pa11y is built on top of html sniffer

**Comment by tnguyen14** [comment]
I see. Good to know. Thanks.
